### DISCUSSION TITLE
169

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#130] [General]
# 潮流周刊第169期 - 路边繁花

封面图来源于周末在市区路边的一个长满花的树，可谓繁花，配上一个大大的突兀摄像头，居然有那么一点点行为艺术的感觉。

https://weekly.tw93.fun/posts/169-%E8%B7%AF%E8%BE%B9%E7%B9%81%E8%8A%B1

**Comment by hesetiema** [comment]
牛奶那里有个错别字👀

**Reply by tw93** [reply]
哪儿，我看了一遍，居然没有发现

**Reply by ricardoo9** [reply]
打奶泡搭配咖啡或者单独“和”，应该是喝

**Reply by tw93** [reply]
哈哈 好的 谢谢指出！

**Comment by Beeta** [comment]
看陈皓那篇文章，中间有一段：「四十岁时，经历过的事太多，发现留给自己的时间不多，世界太复杂，而还有好多事没做，从而变得与世无争，也变得更为地自我。」，再看看文章的日期19年，谁能想到四年后，他就离我们而去了…

**Reply by tw93** [reply]
珍惜当下

**Comment by xxholly32** [comment]
现在日子好了，但大家反而更焦虑了

**Reply by tw93** [reply]
可能还是机会少人多接不住的缘故，心态当然也很重要
